plugin-server/README.md

```diff
@@ -85,46 +85,47 @@ If `PLUGIN_SERVER_MODE` is not set the plugin server will execute all of its tas
+| STATSD_PREFIX                              | StatsD prefix                                                                                                                                                                                                  | `'plugin-server.'`                    |
+| DISABLE_MMDB                               | whether to disable MMDB IP location capabilities                                                                                                                                                               | `false`                               |
+| INTERNAL_MMDB_SERVER_PORT                  | port of the internal server used for IP location (0 means random)                                                                                                                                              | `0`                                   |
+| DISTINCT_ID_LRU_SIZE                       | size of persons distinct ID LRU cache                                                                                                                                                                          | `10000`                               |
+| PISCINA_USE_ATOMICS                        | corresponds to the piscina useAtomics config option (https://github.com/piscinajs/piscina#constructor-new-piscinaoptions)                                                                                      | `true`                                |
+| PISCINA_ATOMICS_TIMEOUT                    | (advanced) corresponds to the length of time (in ms) a piscina worker should block for when looking for tasks - instances with high volumes (100+ events/sec) might benefit from setting this to a lower value | `5000`                                |
+| HEALTHCHECK_MAX_STALE_SECONDS              | 'maximum number of seconds the plugin server can go without ingesting events before the healthcheck fails'                                                                                                     | `7200`                                |
+| MAX_PENDING_PROMISES_PER_WORKER            | (advanced) maximum number of promises that a worker can have running at once in the background. currently only targets the exportEvents buffer.                                                                | `100`                                 |
+| KAFKA_PARTITIONS_CONSUMED_CONCURRENTLY     | (advanced) how many kafka partitions the plugin server should consume from concurrently                                                                                                                        | `1`                                   |
+| RECORDING_PARTITIONS_CONSUMED_CONCURRENTLY | (advanced) how many kafka partitions the recordings consumer should consume from concurrently                                                                                                                  | `1`                                   |
```

Comment:
Should this default value be 5?

Reply:
I mean, should the value in the README be updated to 5 (which looks like the default).